language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/DamlLedgerClient.java

```diff
@@ -147,6 +151,7 @@ private DamlLedgerClient(@NonNull NettyChannelBuilder channelBuilder, @NonNull O
     public DamlLedgerClient(Optional<String> expectedLedgerId, @NonNull ManagedChannel channel) {
         this.channel = channel;
         this.expectedLedgerId = expectedLedgerId.orElse(null);
+        this.accessToken = Optional.empty();
```

Comment:
Do you think it's worth capturing this in a test?

Reply:
(Not) addressed by https://github.com/digital-asset/daml/pull/3793
This constructor is deprecated and our Scala tests are compiled with fatal warning (which unfortunately cannot be adjusted to whitelist given warnings).